league_rpc_linux/lobby.py

```diff
@@ -0,0 +1,36 @@
+from league_rpc_linux.lcu.LCU import getLcuAPI
+
+lcu = getLcuAPI()
+
+
+def checkLcuAPI():
+    print(lcu.getDataFromPath("/lol-summoner/v1/current-summoner").items())
+
+
+def inClient():
```

Comment:
Always add type hints. in parameters, and in return types.

Reply:
invalid, as not existing anymore.